graphql/util.go

```diff
@@ -1290,3 +1290,67 @@ func getAPISubscriptionsForProject(ctx context.Context, projectId string) ([]*re
 	}
 	return res, nil
 }
+
+// GroupProjects takes a list of projects and groupes them by their repo. If onlyDefaultedToRepo is true,
+// it only groups projects that are defaulted to the repo
+func GroupProjects(projects []model.ProjectRef, onlyDefaultedToRepo bool) ([]*GroupedProjects, error) {
+	groupsMap := make(map[string][]*restModel.APIProjectRef)
+
+	for _, p := range projects {
+		groupName := fmt.Sprintf("%s/%s", p.Owner, p.Repo)
+
+		if onlyDefaultedToRepo && p.UseRepoSettings {
+			groupName = ""
+		}
```

Comment:
This would change the group name we show on the project select dropdown. Is this new name appropriate/ something we should show to all the users?

Reply:
right now it shows "/". I'm not a fan of that either. I figured I'll leave it blank for now and we can decide when we build the UI.